### PR TITLE
Check for Kube Object Sync time before running failover for Discovered Apps

### PR DESCRIPTION
**Pull request by prsurve**
…d Apps

### CODE REVIEW

**Review by am-agrawa — APPROVED** [review]

File: ocs_ci/helpers/dr_helpers.py [hunk 1]
```diff
@@ -1661,3 +1661,49 @@ def generate_kubeobject_capture_interval():
         return 5
     else:
         return capture_interval
+
+
+def verify_last_kubeobject_protection_time(drpc_obj, kubeoject_sync_interval):
```
[the remark this answers is not in the document]
Comment (prsurve) [reply ocs_ci/helpers/dr_helpers.py]: Done

File: tests/functional/disaster-recovery/regional-dr/test_failover_and_relocate_discovered_apps.py
[the remark this answers is not in the document]
Comment (prsurve) [reply tests/functional/disaster-recovery/regional-dr/test_failover_and_relocate_discovered_apps.py:1]: done

File: tests/functional/disaster-recovery/regional-dr/test_failover_and_relocate_discovered_apps.py [hunk 2]
```diff
@@ -87,11 +95,17 @@ def test_failover_and_relocate_discovered_apps(self, discovered_apps_dr_workload
...
             rdr_workload.workload_namespace, discovered_apps=True
         )
+
         logger.info("Running Relocate Steps")
         wait_time = 2 * scheduling_interval  # Time in minutes
         logger.info(f"Waiting for {wait_time} minutes to run IOs")
         sleep(wait_time * 60)
 
+        logger.info("Checking for lastKubeObjectProtectionTime")
+        dr_helpers.verify_last_kubeobject_protection_time(
```
[the remark this answers is not in the document]
Comment (prsurve) [reply tests/functional/disaster-recovery/regional-dr/test_failover_and_relocate_discovered_apps.py:105]: yes you are right

**Review by am-agrawa — DISMISSED** [review]

File: ocs_ci/helpers/dr_helpers.py [hunk 3]
```diff
@@ -1715,3 +1715,49 @@ def verify_drpc_deletion(cmd, expected_output_lst):
...
+    Args:
+        drpc_obj (obj): DRPC object
+        kubeobject_sync_interval (int): The KubeObject sync interval in minutes
+
+    Returns:
+        str: Current lastKubeObjectProtectionTime
+
+    Raises:
+        AssertionError: If the lastKubeObjectProtectionTime is outside the expected range
+            (greater than or equal to three times the scheduling interval)
```
[the remark this answers is not in the document]
Comment (prsurve) [reply ocs_ci/helpers/dr_helpers.py]: done

**Review by keesturam — APPROVED** [review]

**Pull request merged**